picamera2/picamera2.py

```diff
@@ -268,8 +276,8 @@ def preview_configuration(self, main={}, lores=None, raw=None, transform=libcame
         lores = self.make_initial_stream_config({"format": "YUV420", "size": main["size"]}, lores)
         raw = self.make_initial_stream_config({"format": self.sensor_format, "size": main["size"]}, raw)
         # Let the framerate vary from 12fps to as fast as possible.
-        controls = {"NoiseReductionMode": libcamera.NoiseReductionMode.Minimal,
-                    "FrameDurationLimits": (self.camera_controls["FrameDurationLimits"][0], 83333)} | controls
+        controls = {"NoiseReductionMode": libcamera.controls.draft.NoiseReductionModeEnum.Minimal,
+                    "FrameDurationLimits": (self.camera_ctrl_info["FrameDurationLimits"][1].min, 83333)} | controls
```

Comment:
I'd like to do something for our users about this nasty long `libcamera.controls.draft` thing, but that's probably for a subsequent commit!